style/style.css

```diff
@@ -1,49 +1,51 @@
 body {
-  font-family: 'Lato', sans-serif;
+  font-family: "Lato", sans-serif;
   color: #272a31;
 }
+
```

Comment:
- [ ] You have a white space in all sections on the page. Please, be kindly to fix it. It will make your project for pretty view and professional for view. Hint: You can just add into your `CSS` file the style for your `body` tag, like below: 
```
margin: 0;
padding: 0;
box-sizing: border-box;
```